benchmark/sql_regressions/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,53 @@
+pipeline {
+    agent none
+    stages {
+        stage('Test Dolt nightly') {
+            parallel {
+                stage ("Update Liquidata/dolt-sql-performance:nightly") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        DOLT_VERSION = sh(returnStdout: true, script: "(git rev-parse HEAD | tr -d '\n')")
+                        TMPDIR = "${pwd()}/tempDir"
+                        DOLT_ROOT_PATH="${pwd()}/tempRoot"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        DOLT_GLOBAL_CONFIG = credentials("system-account-dolthub-config")
+                    }
+                    steps {
+                        sh "rm -rf $TMPDIR && mkdir $TMPDIR"
+                        sh "rm -rf $DOLT_ROOT_PATH && mkdir $DOLT_ROOT_PATH"
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("go") {
```

Comment:
Don't do this if the script is just going to build/install dolt which it needs...